### ISSUE TITLE
var BUTTON not working

### ISSUE THREAD

**Issue by dimitre** [plotdevice/plotdevice#44]
This is the minimum code to reproduce the issue. Tested and running OK in Nodebox
Doesn't fire event in PlotDevice

```python
var("comando", BUTTON)

def comando():
    print "save"
```

**Comment by gnuyens** [comment]
Hi folks, I've really been enjoying PlotDevice.  However, this var bug (and a separate problem with the Colors library which I'll submit separately) is slowing down progress.

It seems like there are no new comments in issues for quite a while.  Is the project defunct? (not meant as a flame, just a practical question).

Thanks
Greg

**Comment by samizdatco** [comment]
fixed in: a391cef3425e4618b220f22e2a7dcd8660e0678e

The usage for buttons looks like this:
```python
var(function_name, BUTTON, button_text, color=None, label=None)
```

At its simplest, code like this:
```python
def do_something():
  print("i did it")

var('do_something', BUTTON)
```
will produce a floating palette with a button named after your callback function like:
<img width="305" alt="image" src="https://user-images.githubusercontent.com/469523/178157446-496e84e6-318a-4c5c-bd01-5045655f1af0.png">

You can customize both the text on the button itself and the 'prompt' string to its left:
```python
var('do_something', BUTTON, "Go!", label="Just say when")
```
<img width="343" alt="image" src="https://user-images.githubusercontent.com/469523/178157554-263ae7ac-9c12-4b3f-8bbe-1628be35c1f6.png">

And if you want to change the button color, you can pass either a Color object or valid arguments for the current color mode like:
```python
var('do_something', BUTTON, "Go!", label="Just say when", color='#f60')
```
or
```python
var('do_something', BUTTON, "Go!", label="Just say when", color=(1, .5, 0))
```
<img width="343" alt="image" src="https://user-images.githubusercontent.com/469523/178157637-d2acfaef-a64f-4b45-bd84-9d7ad7861737.png">

**Comment by dimitre** [comment]
outstanding news seeing your activity here @samizdatco 
its my favorite fork keep up with the good work.
